client/mysqldump.c

```diff
@@ -4130,9 +4175,7 @@ static void dump_table(const char *table, const char *db, const uchar *hash_key,
   init_dynamic_string_checked(&query_string, "", 1024, 1024);
 
   if (path)
-  {
-    char filename[FN_REFLEN], tmp_path[FN_REFLEN];
```

Comment:
was there a need to move this?

Reply:
Actually not, thank you for your remark. This move remained unnoticed from a previous coding experience. I am going to move it back to its original place.

Reply:
Done